djangoproject/templates/aggregator/ecosystem.html

```diff
@@ -139,7 +139,7 @@ <h3 id="tutorials">
   <ul>
     <li>
       <a href="https://tutorial.djangogirls.org/">DjangoGirls Tutorial</a>
-      &mdash; A great tutorial you can do in addition to the <a href="{% url 'document-detail' lang='en' version='stable' url='intro/tutorial/01/' host 'docs' %}">official tutorial</a>.
+      &mdash; A great tutorial you can do in addition to the <a href="{% url 'document-detail' lang='en' version='stable' url='intro/tutorial01/' host 'docs' %}">official tutorial</a>.
```

Comment:
```suggestion
      &mdash; A great tutorial you can do in addition to the <a href="{% url 'document-detail' lang='en' version='stable' url='intro/tutorial01' host 'docs' %}">official tutorial</a>.
```
This also has an extra url slash 👍